### ISSUE TITLE
about style domain adaptation

### ISSUE THREAD

**Issue by sezarxray** [yorkeyao/VehicleX#11]
Hi, what a nice project!
Could you plz tell me more about training data for style domain adaptation?
what is trainA/B?
what is testA/B?
How many data we need?

    │   └───sys2real
    |   │   │ trainA
    |   │   │ trainB
    |   │   │ testA
    |   │   │ testA

Thanks in advance

**Comment by yorkeyao** [comment]
Thank you for your question. 

For training, if you want to perform a style transfer from VehicleX to VeRi. VehicleX is trainA and the training set of VeRi is trainB. Then you will get a model Gb to generate VeRi style VehicleX data.   

In my setting, I may not really need a test set. Thank you for pointing this out and I have modified the README. You may need to modify the args source_path and save_path in test_spgan.py to inference. In this case, the source_path is the path to VehicleX data.

**Comment by sezarxray** [comment]
How many data we need?

**Comment by yorkeyao** [comment]
I use the entire VehicleX training set and entire VeRi training set.

**Comment by sezarxray** [comment]
thx
